src/Web64/Nlp/NlpClient.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\strawberry_runners\Web64\Nlp;
+
+use Web64\Nlp\NlpClient as Web64NlpClient;
+
+class NlpClient extends Web64NlpClient {
+
+  const MAX_RETRY_HOST = 3;
+
+  /**
+   *  Get language code for text
+   *
+   * @param     $text
+   * @param int $predictions
+   *
+   * @return mixed|null
+   */
+  public function fasttext($text, $predictions = 1) {
+    //Predictions at least 1 are required if POSTING. GET is more forgiving
```

Comment:
Ah - I think I understand the problem. Predictions should be set to 1 if the parameter is not supplied (e.g as in a GET request with only `?text=xyz` as the querystring): https://github.com/digitaldogsbody/nlpserver-fasttext/blob/master/nlpserver.py#L501-L502, but because I modeled the code after the existing endpoints, when the parameter is supplied but empty, or with an invalid value (e.g a POST where formdata is built), it just returns an error: https://github.com/digitaldogsbody/nlpserver-fasttext/blob/master/nlpserver.py#L504-L510

I can fix this very easily to always assume minimum 1 rather than erroring (this is what I would have done if I'd been writing the NLPServer from scratch - and should be expected behaviour in the rewrite)